<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCourse</class>
 <widget class="QDialog" name="addCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0,0">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>添加课程</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>课程名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_course_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>课程类别:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_course_type">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>必修理论课程</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>选修理论课程</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>实验课程</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>体育课程</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>建议修读学期:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_term">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>周几:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_course_day">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期三</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期四</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期五</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期六</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>第几节:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_course_time">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7-8</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>教师:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_teacher"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>地点:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_location"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>容量:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_num"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>学分:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_credit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
